### PR TITLE
[Documentation][Cookbook]Adding validation for image uploads

### PR DESCRIPTION
**Pull request by Mipme**
Adding description on how to validate the images that can be uploaded when finishing this guide.

| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes none
| License         | MIT

### CODE REVIEW

File: docs/cookbook/images/images-on-entity.rst [hunk 1]
```diff
@@ -395,6 +395,41 @@ and render the ``{{ form_row(form.images) }}`` field.
...
+              mimeTypes: 
+                - "image/png"
+                - "image/jpg"
+                - "image/jpeg"
+                - "image/gif"
+              mimeTypesMessage: 'This mime is not allowed. Pls. use PNG, JPG or GIF files.'
+              minHeight: 200
+              minWidth: 200
+              
+This defines the validation constraints for each image entity. Now connecting the validation of the ShippingMethod to the validation of each single Image Entity is left:
```
Comment (CoderMaggie) [inline docs/cookbook/images/images-on-entity.rst]: can we break this line? it's a bit too long :)

File: docs/cookbook/images/images-on-entity.rst [hunk 2]
```diff
@@ -395,6 +395,41 @@ and render the ``{{ form_row(form.images) }}`` field.
...
+              groups: [sylius]
+              maxHeight: 1000
+              maxSize: 10240000
+              maxWidth: 1000
+              mimeTypes: 
+                - "image/png"
+                - "image/jpg"
+                - "image/jpeg"
+                - "image/gif"
+              mimeTypesMessage: 'This mime is not allowed. Pls. use PNG, JPG or GIF files.'
```
Comment (CoderMaggie) [inline docs/cookbook/images/images-on-entity.rst]: `Pls.` -> `Please`
`This mime` -> `This file format`

**Comment by CoderMaggie** [comment]
Can you please rebase your PR with the `1.0` branch as it an enhancement to existing docs? :)

File: docs/cookbook/images/images-on-entity.rst [hunk 3]
```diff
@@ -395,6 +395,42 @@ and render the ``{{ form_row(form.images) }}`` field.
 
     Learn more about customizing templates :doc:`here </customization/template>`.
 
+13. Validation
+^^^^^^^^^^^^^^
+
+Your form so far is working fine, but don't forget about validation.
+The easiest way is using validation config files under the ``AppBundle/Resources/config/validation`` folder.
+
+This could look like this f.i.:
```
Comment (pamil) [inline docs/cookbook/images/images-on-entity.rst]: `f.i.` -> `e.g.`?

File: docs/cookbook/images/images-on-entity.rst [hunk 4]
```diff
@@ -395,6 +395,42 @@ and render the ``{{ form_row(form.images) }}`` field.
...
+                - "image/png"
+                - "image/jpg"
+                - "image/jpeg"
+                - "image/gif"
+              mimeTypesMessage: 'This file format is not allowed. Please use PNG, JPG or GIF files.'
+              minHeight: 200
+              minWidth: 200
+              
+This defines the validation constraints for each image entity.
+Now connecting the validation of the ``ShippingMethod`` to the validation of each single ``Image Entity`` is left:
```
Comment (pamil) [inline docs/cookbook/images/images-on-entity.rst]: I'm missing the information what file should I modify here.

**Comment by pamil** [comment]
Thanks @Mipme! 🥇

**Pull request merged**